tests/layer_tests/tensorflow_tests/test_tf_BatchMatrixInverse.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestBatchMatrixInverse(CommonTFLayerTest):
+    def prepare_input(self, inputs_info):
```

Comment:
please set statically defined input matrices that are more practicable and non-singular. Check https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/tensorflow_tests/test_tf_MatrixInverse.py as a reference how to do.

Reply:
static matrices with float values have been implemented